index.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio skeleton</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" 
+  integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" 
+  crossorigin="anonymous">  
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio_Website</title>
 </head>
 <body>
-    <h1>Designing Portfolio Skeleton</h1>
+  <!--Header Section -->
+
+  <main class="head-section"> 
+      <nav class="top d-flex">
+        <h1>My Logo</h1>
+        <i class="fas fa-bars"></i>
+      </nav>     
+
+     <!--Main Section -->
+
+     <section  class="main-section">
+      <h2 class="intro">I'm Ravi Teja<span>glad to see</span><span>
```

Comment:
- [ ] As I sad before the the main title(your name) should be a` <h1>` and not a `<h2>` tag. use this
```
<h1 class="intro">I'm Ravi Teja<span>glad to see</span><span>
 you!!</span></h1>
```